Tasks/Xcode/Tests/L0CreateIpaWithCodeSigningIdentifiers.ts

```diff
@@ -53,9 +53,6 @@ let a: ma.TaskLibAnswers = <ma.TaskLibAnswers>{
     "getVariable": {
```

Comment:
I know you didn't write this, but this is something I noticed and would like to clean up as we touch these files:
This isn't doing anything, since `getVariable` isn't mocked.  It isn't on the `TaskLibAnswers` interface, but the compiler doesn't catch it because we are coercing the type.
We can just pass the object literal as the argument to `setAnswers` without using the the one-time variable.  Then the compiler will catch it.

Reply:
The original plan was to eventually mock getVariable, so we set the variable via the process and also setup the mock.

Reply:
Sorry, after looking at it further it looks like we are coercing it because `TaskLibAnswers` is not up-to-date.  It is the case that `getVariable` is not mocked and is returning the real value from `process.env`.  However, other functions such as `stats` are mocked but are not on the interface, and type-checking the interface will prevent us from using these.  We should update `TaskLibAnswers` upstream at some point to bring it up-to-date and then make these changes.